python/ray/dag/compiled_dag_node.py

```diff
@@ -661,7 +661,7 @@ def __init__(
             # We conservatively set num_shm_buffers to _max_inflight_executions.
             # It means that the DAG can be underutilized, but it guarantees there's
             # no false positive timeouts.
-            num_shm_buffers=self._max_inflight_executions,
+            num_shm_buffers=1,
```

Comment:
Can you update the comment above and add a TODO?

Reply:
I will fix this this week, so it may be okay?

Reply:
Yes, feel free to merge first if urgent.